packages/discord.js/src/structures/StringSelectMenuBuilder.js

```diff
@@ -24,8 +24,8 @@ class StringSelectMenuBuilder extends BuildersSelectMenu {
 
   /**
    * Normalizes a select menu option emoji
-   * @param {SelectMenuOptionData|APISelectMenuOption} selectMenuOption The option to normalize
-   * @returns {SelectMenuOptionBuilder|APISelectMenuOption}
+   * @param {StringSelectMenuOptionData|APISelectMenuOption} selectMenuOption The option to normalize
```

Comment:
The first of these types doesn't exist, and never seemed to? Can't find anything easily in history to figure out what it was.